From fbf5d2c2568dace1b05a370229b52d8e3315d460 Mon Sep 17 00:00:00 2001
From: Sheng Nan <b38800@freescale.com>
Date: Fri, 1 Mar 2013 10:30:02 +0800
Subject: [PATCH 2402/2463] ENGR00252071-3: mxc_v4l2_capture: enable auto
 detect of ov5642/5640 in sabresd

- enable the auto detect for parallel ov5642 and ov5640 in sabresd board.

Signed-off-by: Sheng Nan <b38800@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabresd.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabresd.c b/arch/arm/mach-mx6/board-mx6q_sabresd.c
index 986d291..082de98 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabresd.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabresd.c
@@ -782,7 +782,7 @@ static struct i2c_board_info mxc_i2c0_board_info[] __initdata = {
 		I2C_BOARD_INFO("wm89**", 0x1a),
 	},
 	{
-		I2C_BOARD_INFO("ov5642", 0x3c),
+		I2C_BOARD_INFO("ov564x", 0x3c),
 		.platform_data = (void *)&camera_data,
 	},
 	{
-- 
1.7.7.4

